docs/includes/fundamentals/write-operations/WriteOperationsTest.php

```diff
@@ -329,4 +330,188 @@ public function testModelPositional(): void
+                'venue' => 'Soldier Field',
+                'genres' => [ 'rock', 'pop', 'blues' ],
+                'ticketsSold' => 59527,
+                'performanceDate' => new UTCDateTime(Carbon::create(2024, 6, 30, 20, 0, 0, 'CDT')),
+            ],
+        ];
+        Concert::insert($data);
+
+        $this->assertEquals(4, Concert::count());
+
+        $id = Concert::first()->id;
+
+        // begin model delete by id
+        Concert::destroy($id);
+        // end model delete by id
+
+        $this->assertEquals(3, Concert::count());
+    }
+
+    public function testModelDeleteModel(): void
+    {
+        require_once __DIR__ . '/Concert.php';
+        Concert::truncate();
+
+        $data = [
+            [
+                'performer' => 'Mitsuko Uchida',
+                'venue' => 'Carnegie Hall',
+            ],
+        ];
+        Concert::insert($data);
+
+        // begin delete one model
+        $concert = Concert::first();
+        $concert->delete();
+        // end delete one model
+
+        $this->assertEquals(0, Concert::count());
+    }
+
+    public function testModelDeleteFirst(): void
+    {
+        require_once __DIR__ . '/Concert.php';
+        Concert::truncate();
+
+        $data = [
+            [
+                'performer' => 'Mitsuko Uchida',
+                'venue' => 'Carnegie Hall',
+            ],
+            [
+                'performer' => 'Brad Mehldau',
+                'venue' => 'Philharmonie de Paris',
+            ],
+            [
+                'performer' => 'Billy Joel',
+                'venue' => 'Madison Square Garden',
+            ],
+            [
+                'performer' => 'The Rolling Stones',
+                'venue' => 'Soldier Field',
+            ],
+        ];
+        Concert::insert($data);
+
+        // begin model delete one fluent
+        Concert::where('venue', 'Carnegie Hall')
+            ->orderBy('_id')
+            ->limit(1)
+            ->delete();
+        // end model delete one fluent
+
+        $this->assertEquals(3, Concert::count());
+    }
+
+    public function testModelDeleteMultipleById(): void
+    {
+        require_once __DIR__ . '/Concert.php';
+        Concert::truncate();
+        $data = [
+            [
+                'performer' => 'Mitsuko Uchida',
+                'venue' => 'Carnegie Hall',
+            ],
+            [
+                'performer' => 'Brad Mehldau',
+                'venue' => 'Philharmonie de Paris',
+            ],
+            [
+                'performer' => 'Billy Joel',
+                'venue' => 'Madison Square Garden',
+            ],
+            [
+                'performer' => 'The Rolling Stones',
+                'venue' => 'Soldier Field',
+            ],
+        ];
+        Concert::insert($data);
+
+        $concerts = Concert::all();
+
+        $ids = [];
+
+        foreach ($concerts as $concert) {
+            array_push($ids, $concert->id);
+        }
```

Comment:
Laravel has [`Collection::pluck()` method](https://laravel.com/docs/11.x/collections#method-pluck) to do that:

```suggestion
        $concerts = Concert::all()->pluck('_id')->all();
```